internal/provider/column_resource.go

```diff
@@ -57,7 +57,7 @@ func (r *columnResource) Configure(_ context.Context, req resource.ConfigureRequ
 	r.client = c
 
 	features, err := w.Features()
-	if err != nil || features == nil {
```

Comment:
this was silly: the underlying function returns error if features is nil already and even if it was nil it would panic as it calls `err.Error()` 🙃